utils/calendrical_calculations/src/astronomy.rs

```diff
@@ -234,27 +242,27 @@ impl Astronomical {
 
         if (2051..=2150).contains(&year_int) {
             (-20.0
-                + 32.0 * libm::pow((year_int as f64 - 1820.0) / 100.0, 2.0)
-                + 0.5628 * (2150.0 - year_int as f64))
+                + 32.0 * (((year_int - 1820) * (year_int - 1820)) as f64 / 10000.0)
```

Comment:
Note: Andrew found that calling pow was faster than calling multiply in some of these cases, which I found surprising, but he has data

Reply:
do you know if he compared to multiplying floats or integers? I'd be surprised if the floating point operations here are cheaper than the integer operations and one float division.